assets/js/modules/thank-with-google/datastore/publications.js

```diff
@@ -155,6 +162,39 @@ const baseSelectors = {
 			publications[ 0 ]
 		);
 	} ),
+
+	/**
+	 * Gets the link to create new publication.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {string} Create publication URL.
+	 */
+	getServiceCreatePublicationURL: createRegistrySelector(
+		( select ) => () => {
+			const homeURL = select( CORE_SITE ).getHomeURL();
+
+			const url = new URL( 'https://publishercenter.google.com/' );
+			url.searchParams.set( 'sk_url', encodeURIComponent( homeURL ) );
+
+			return select( CORE_USER ).getAccountChooserURL( url.toString() );
+		}
+	),
+
+	/**
+	 * Gets the link of an existing publication.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} _state        Data store's state.
+	 * @param {string} publicationID The ID of the publication to get link for.
+	 * @return {string} Publication link.
+	 */
+	getServicePublicationURL: ( _state, publicationID ) => {
+		invariant( publicationID, 'A publicationID is required.' );
+
+		return `https://publishercenter.google.com/publications/${ publicationID }/overview`;
```

Comment:
We should probably encode the `publicationID` value with the `encodeURIComponent` function.

Reply:
Addressed this, thank you!